tracer/test/test-applications/regression/TransportsTester/TransportsTester.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+  </PropertyGroup>
```

Comment:
Same question, is this required for tests? If not, can we exclude it from the solution and the build so it doesn't impact CI etc?

Reply:
It is used in 
```
        public MetricsTransportTests(ITestOutputHelper output)
            : base("TransportsTester", @"test\test-applications\regression", output, prependSamplesToAppName: false)
        {
        }```
        
        But I can probably use another general purpose sample for this.
        This `TransportsTester` project was what I originally used in conjunction with the `MockAgent` project you also poo-poo'ed.

Reply:
"poo-poo'ed" is a bit strong 😆 I'm just concerned about our ever-climbing build times!

Reply:
I removed it and piggy backed on current runtime metrics tests